tsconfig.json

```diff
@@ -1,10 +1,6 @@
 {
+  "extends": ["@tsconfig/node18/tsconfig.json"],
   "compilerOptions": {
-    "lib": ["ES2022"],
-    "target": "ES2022",
-    "module": "commonjs",
-    "esModuleInterop": true,
-    "moduleResolution": "node",
     "allowJs": true,
     "checkJs": true,
     "noEmit": true,
```

Comment:
nit: can also remove line 7 `"strict": true `, as it is already in [@tsconfig/node18](https://www.npmjs.com/package/@tsconfig/node18)